docs/aws-sagemaker-hosted-scorer.md

```diff
@@ -0,0 +1,152 @@
+# Driverless AI Deployment Template for AWS Sagemaker Hosted Scorer
+
+This [template](https://github.com/h2oai/dai-deployment-templates/tree/master/aws-sagemaker-hosted-scorer) scores realtime data using Driverless AI Mojo pipeline with Java Runtime and plugs into the AWS SageMaker workflow documented [here](https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-inference-code.html)
+
+It is a REST API which accepts one data point at a time for prediction in real-time in the hosted SageMaker environment. The user needs to provide Driverless AI license key and  model's  pipeline.mojo file for scoring, see [deploy](###deploy-to-sageMaker) section for details. The versions of software used to create the template like mojo runtime are listed [here](https://github.com/h2oai/dai-deployment-templates/blob/master/gradle.properties#L8).
```

Comment:
@NidhiMehta what is your thinking - the MOJO runtime supports not only DAI models, but also OSS models. Should we mention it here? For example, have a dedicated page of supported models - DAI, H2O-3, H2O-3 Klime.